roles/redpanda_broker/tasks/install-rp-rpm-airgap.yml

```diff
@@ -2,12 +2,20 @@
   ansible.builtin.copy:
     src: "{{ airgap_copy_src }}/redpanda_rpms.tar.gz"
     dest: "{{ airgap_copy_dest }}/redpanda_rpms.tar.gz"
+  tags:
+    - airgap-tarball-install
 
 - name: Unpack Redpanda RPMs on host
   ansible.builtin.unarchive:
     src: "{{ airgap_copy_dest }}/redpanda_rpms.tar.gz"
     dest: "{{ airgap_copy_dest }}"
+  tags:
+    - airgap-tarball-install
 
 - name: Install Redpanda RPMs
-  ansible.builtin.shell: "rpm -Uvh {{ airgap_copy_dest }}/*.rpm"
+  ansible.builtin.shell: "rpm -Uvh --force {{ airgap_copy_dest }}/*.rpm"
```

Comment:
ansible.builtin.yum can take a local path to a package and handle direct installation without needing the exec hacks here (or error checking) if I recall correctly.

Reply:
> You can also pass a url or a local path to a rpm file (using state=present). To operate on several packages this can accept a comma separated string of packages or (as of 2.0) a list of packages.

See comment under the `name` option